### ISSUE TITLE
null value in pointerLabelComponent inside pointerConfig 

### ISSUE THREAD

**Issue by dumihi313** [Abhinandan-Kushwaha/react-native-gifted-charts#752]
<img width="747" alt="image" src="https://github.com/user-attachments/assets/b36acdc4-266a-464d-a07d-22aca7bec185">
The recent version return null value,  I am using dataset with secondary items

**Comment by Abhinandan-Kushwaha** [comment]
Hi @dumihi313 👋

Thanks for reporting this issue. It has been fixed from version `1.4.25` onwards. 🎉
Please use the latest version of the library.

**Comment by Deepakgullak** [comment]
Hi @Abhinandan-Kushwaha, cannot share complete code but something similar to this, i am storing data in an array, where i push value along with some other keys, all the other keys are giving correct after hover complete, but for some reason the `value` prop is showing undefined when the hover is done
```
              <LineChart
                curved
                areaChart
                adjustToWidth={true}
                data={data}
                thickness={4}
                isAnimated={true}
                hideAxesAndRules={true}
                hideOrigin={true}
                showDataPointOnFocus={true}
                areaGradientId="ag"
                areaGradientComponent={() => {
                  return (
                    <SvgLinearGradient id="ag" x1="201.053" y1="201.19" x2="213" y2="10.2521" gradientUnits="userSpaceOnUse">
                      <Stop stopColor={'#FFFADF'} stopOpacity={0} />
                      <Stop offset="0.40625" stopColor={'#FFEDA3'} />
                    </SvgLinearGradient>
                  );
                }}
                pointerConfig={{
                  pointerStripUptoDataPoint: true,
                  autoAdjustPointerLabelPosition: true,
                  pointerStripColor: '#236561',
                  pointerStripWidth: 4,
                  pointerLabelWidth: 140,
                  pointerLabelHeight: 100,
                  pointerComponent: () => {
                    return <View style={styles.pointerComponent} />;
                  },
                  pointerLabelComponent: (items: any) => {
                    const value = NumberTrunc2(items[0]?.quantity * items[0]?.futureGoldRate);
                    return (
                      <View style={Platform.OS === 'android' ? styles.labelComponent : styles.labelComponentIos}>
                        <Text style={styles.toolTipAmountText}>₹{format({items?.[0]?.value})}</Text>
                        <Text style={styles.toolTipQuantityText}>({items?.[0]?.key2})</Text>
                        <Text style={styles.toolTipDateText}>{items?.[0]?.key3}</Text>
                      </View>
                    );
                  },
                }}
              />
```

**Comment by Abhinandan-Kushwaha** [comment]
Hi @Deepakgullak Thanks for reporting this bug. It has been fixed from versions `1.4.34` onwards.
Please use the latest version of the library.

**Comment by Deepakgullak** [comment]
Thanks @Abhinandan-Kushwaha
